src/lib/xapi.ts

```diff
@@ -150,6 +150,7 @@ export const Verb = {
 	PlayedVideo: 'https://w3id.org/xapi/video/verbs/played',
 	Rated: 'http://id.tincanapi.com/verb/rated',
 	Registered: 'http://adlnet.gov/expapi/verbs/registered',
+	Retrieved: 'https://w3id.org/xapi/dod-isd/verbs/retrieved',
```

Comment:
As we're no longer using it lets get rid of `Retrieved` from here, below and in the template.